frame/system/src/lib.rs

```diff
@@ -232,6 +232,19 @@ pub mod pallet {
 			type BlockWeights = ();
 			type BlockLength = ();
 			type DbWeight = ();
+			#[verbatim]
+			type RuntimeEvent = ();
+			#[verbatim]
+			type RuntimeOrigin = ();
+			#[verbatim]
+			type RuntimeCall = ();
+			#[verbatim]
+			type Block = ();
```

Comment:
This is not auto-generated by anyone, should it really be verbatim? 

How will the error message be, if `Block` is not found in scope? We should try and make this as fool proof as possible, in light of what @xlc also pointed out.

Reply:
I have added a UI test to showcase the error. It will be similar to this one: 

```
error[E0412]: cannot find type `Area` in this scope
```